postmaster/static/js/utils.js

```diff
@@ -51,20 +51,25 @@ function changePage(obj, e) {
 // This manages the loading spinner for the table
 function manageSpinner(present) {
 
-    var spinner = $('div.loader');
+    var spinner = $('div.spinner');
+    var spinnerParent = $('div.spinnerParent');
 
     if (present) {
-        if (spinner.length == 0) {
-            $('#dynamicTableDiv').prepend('<div class="loader"></div>');
+        // If the spinner isn't there, create it
+        if (spinnerParent.length == 0) {
+            var spinnerParentHtml = $('<div/>', {'class': 'spinnerParent'});
+            var spinnerHtml = $('<div/>', {'class': 'spinner'});
+            spinnerHtml.append($('<div/>', {'class': 'bounce1'}));
+            spinnerHtml.append($('<div/>', {'class': 'bounce2'}));
+            spinnerHtml.append($('<div/>', {'class': 'bounce3'}));
```

Comment:
Ahh, I was really just trying to simplify this part: `spinnerHtml.append($('<div/>',`

It just looked odd to me. But then again I'm not really versed in DOM manipulation all that well.